### PR TITLE
fix: set display-name in image component #2871

### LINKED ISSUE #2871: `displayName` is not set for the component of "@yamada-ui/image"
script: component-display-name
package: @yamada-ui/image

### Description
The component "@yamada-ui/image" uses `forwardRef`, but `displayName` is not set, making it difficult to see the display in [React Developer Tools](https://github.com/facebook/react/tree/main/packages/react-devtools).

### Additional Information
Set `displayName` and `__ui__` after each component definition.

For example, like this.

```tsx
const Component = forwardRef((props, ref) => {
  ...
})

Component.displayName = 'Component'
Component.__ui__ = 'Component'
```

Replace `'component'` with the name of the component. For example, for a `Select` component, it is `'Select'`.

`__ui__` is an identifier required for internal use by Yamada UI. Be sure to set it together with `displayName`.

### PR DESCRIPTION
**Pull request by sonitan0803**
Closes #2871

## Description
The component "@yamada-ui/image" uses forwardRef, but displayName is not set, making it difficult to see the display in [React Developer Tools](https://github.com/facebook/react/tree/main/packages/react-devtools).

## New behavor

Set displayName.

## Is this a breaking change (Yes/No):
No

### CODE REVIEW

File: packages/components/image/src/image.tsx
Comment (kurakke) [inline packages/components/image/src/image.tsx:1]: Thank you for creating the pull request.
I think you should make a similar change to the NativeImage component.

Comment (sonitan0803) [reply packages/components/image/src/image.tsx:1]: @kurakke 
Thank you for confirming.
It looks like forwardRef is not used in NativeImage, but is native-image.tsx the correct target?

```ts
export const NativeImage: FC<NativeImageProps> = ui("img", {

```

Comment (kurakke) [reply packages/components/image/src/image.tsx:1]: Sorry, I misunderstood. I approve it.

**Review by kurakke — APPROVED** [review]

**Review by hirotomoyamada — APPROVED** [review]

**Comment by hirotomoyamada** [comment]
Thanks!

**Pull request merged**